src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -81,15 +81,15 @@
 				@update:open="screenSharingMenuOpen = true"
 				@update:close="screenSharingMenuOpen = false">
 				<!-- Actions button icon -->
-				<Monitor
+				<MonitorOff
```

Comment:
Ah, now I remember why this was changed.

With this patch when:
* audio is on => icon
* audio is off => icon crossed
* video is on => icon
* video is off => icon crossed
* screenshare is on => icon crossed 😕  *nickvergessen hurt itself in confussion*
* screenshare is off => icon 😕  *nickvergessen hurt itself in confussion*

So for screensharing the icon is invers to the mic/video icon and it feels odd if they are next to each others like that.
Maybe inverting the color of the background and icon is a better state representation?

Reply:
or invert screenshare on vs off to align with the others ?

not sure what @nextcloud/designers-talk think then...

Reply:
This is currently the case but not when your screen is not shared. We wouod have to use a crossed out icon for that and it feels weird...

Reply:
So yeah, for audio and video the icon reflects the current state. For screensharing the logic is inverted, but I’d not necessarily say that’s a bad thing.

Here’s possibly better icons, also from Material design:
- "Present to all" https://fonts.google.com/icons?selected=Material+Icons:present_to_all
- "Cancel presentation" https://fonts.google.com/icons?selected=Material+Icons:cancel_presentation

Reply:
Done